_config.yml

```
@@ -1,5 +1,5 @@
 repository: SwedbankPay/developer.swedbankpay.com
-remote_theme: SwedbankPay/swedbank-pay-design-guide-jekyll-theme
+remote_theme: SwedbankPay/swedbank-pay-design-guide-jekyll-theme@test_without_title_changes
```

Can we perhaps create a `1.0.0` tag in the theme repository and pin that instead?

```suggestion
remote_theme: SwedbankPay/swedbank-pay-design-guide-jekyll-theme@1.0.0
```

---

Ye sure